### ISSUE TITLE
discussion: follow only semver-major Node.js releases

### ISSUE THREAD

**Issue by imyller** [imyller/meta-nodejs#24]
Node.js uses semantic versioning and currently meta-nodejs layer maintains recipe for every semver-minor release. This has proven to be problematic.

About semantic versioning @ http://semver.org

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
```

Recently Node.js has released semver-minor updates to LTS moving from 4.2 to 4.3 and soon 4.4. This means that LTS actually is any 4.x. Additionally all previous stable semver-minor releases in 5.x line are left unmaintained urging users to keep up with the latest semver-minor release.

It would be more proper to maintain OpenEmbedded recipes for each server-major (0.10.x, 0.12.x, 4.x, 5.x) release rather than separate one for every semver-minor release.

If there are no objections, I'll remove unmaintained semver-minor releases and continue releasing separate recipes for only semver-major release lines.

**Comment by imyller** [comment]
Request for comments.

/cc @bachp @jmesmon @geomatsi @gsikorski @fladi

**Comment by gsikorski** [comment]
OK. We use a clone of this repo anyway and it is pretty easy to change minor version.

**Comment by bachp** [comment]
For me it would be sufficient to have the LTS (4.x) and Current version (5.x) supported.
So the change you propose is perfectly fine with me.

**Comment by codyps** [comment]
Sounds fine to me.

**Comment by geomatsi** [comment]
Fine by me.

**Comment by imyller** [comment]
Thank you for your feedback. Implemented this change in PR #26
